src/app/components/LinkedData/index.jsx

```diff
@@ -62,7 +62,7 @@ const LinkedData = ({
     mainEntityOfPage,
   };
 
-  if (type === 'article') {
```

Comment:
Just wondering, how was this problem identified? Is it worth considering if we need any additional test coverage to prevent a recurrence?

Reply:
> how was this problem identified?

When double checking #2452 this morning, I also checked an article using https://search.google.com/structured-data/testing-tool/u/0/#url=https%3A%2F%2Fwww.bbc.co.uk%2Fnews%2Farticles%2Fc5ll353v7y9o and saw errors and investigated

I would definitely like some tests to catch this, I'm not sure of the best way of doing so (I don't find snapshots helpful for comparing lumps of json, looking again I've made another mistake with the snapshots on this PR)

Reply:
I've added a couple of assertions to the article metadata e2e tests to check that the article specific keys exist on article pages (which would have caught the current issue)